src/Scene/Quadtree.js

```diff
@@ -50,6 +50,9 @@ define('Scene/Quadtree', [
         this.tileType = type;
         this.minLevel = 2;
         this.maxLevel = 17;
+        this.wgs84TileLayer = null;
+        this.colorLayers = [];
+        this.elevationLayers = [];
```

Comment:
Yeah, they won't last long here.

Currently NodeProcess knows about these layer (e.g: `for (j = 0; j < quadtree.colorLayers.length; j++) {` so if you move them straight to Globe you'll add a dependency between NodeProcess and Globe.

I can do this in this PR, but I'd prefer to get this one merged, and do another PR later to fix this issue.